### PR TITLE
Fix webos white screen

### PR DESCRIPTION
**Pull request by mihaiblaga89**
## Description

- Add babel-preset-env to webpack
- Provide proper polyfills for webos
- Tested on: macos, web with engine rn-web, tizen will be part of https://github.com/flexn-io/renative/issues/1268

## Related issues

- https://github.com/flexn-io/renative/issues/925

## Npm releases

n/a

### CODE REVIEW

File: packages/engine-rn-web/templates/appShell/index.html [hunk 1]
```diff
@@ -1,34 +1,5 @@
...
-
-<head>
-  <meta charset="UTF-8">
-  <title>App Shell</title>
-  <style>
-    body {
-      background-color: white;
-    }
-  </style>
-  {{APPSHELL_HTML_HEADER}}
```
Comment (pavjacko) [inline packages/engine-rn-web/templates/appShell/index.html:11]: @mihaiblaga89 removing this inject APPSHELL_HTML_HEADER also makes config prop `hostedShellHeaders` no longer working. is that intentional?

**Review by pauliusguzas — CHANGES_REQUESTED** [review]
rc.12: all white blank screen
this PR: all black blank screen

so the issue isn't actually fixed

**Review by pauliusguzas — APPROVED** [review]
fixed for webos

File: packages/sdk-webpack/src/config/webpack.config.js [hunk 2]
```diff
@@ -391,12 +391,14 @@ module.exports = function (webpackEnv) {
                             options: {
                                 // customize: require.resolve('babel-preset-react-app/webpack-overrides'),
                                 presets: [
-                                    // [
-                                    //     require.resolve('babel-preset-react-app'),
-                                    //     {
-                                    //         runtime: hasJsxRuntime ? 'automatic' : 'classic',
-                                    //     },
-                                    // ],
+                                    [
```
Comment (pavjacko) [inline packages/sdk-webpack/src/config/webpack.config.js:394]: @mihaiblaga89 can you wrap this with RNV OVERRIDE comments to ensure these changes don't get overridden during future upgrades

**Review by pavjacko — APPROVED** [review]

**Pull request merged**